src/share/classes/com/sun/javatest/regtest/util/ProcessUtils.java

```diff
@@ -27,20 +27,35 @@
 
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.Objects;
 
 /**
  * Utilities for handling Processes.
  */
 public class ProcessUtils {
 
-    private static Method destroyForciblyMethod;
+    private static final Method destroyForciblyMethod;
+    private static final Method PID_METHOD;
+
+    private static final long UNKNOWN_PID = -1;
 
     static {
+        Method destroyMethod;
         try {
-            destroyForciblyMethod = Process.class.getDeclaredMethod("destroyForcibly");
+            destroyMethod = Process.class.getDeclaredMethod("destroyForcibly");
         } catch (NoSuchMethodException e) {
             // expected on pre-1.8 JDKs
+            destroyMethod = null;
         }
+        destroyForciblyMethod = destroyMethod;
+
+        Method pidMethod = null;
+        try {
+            pidMethod = Process.class.getDeclaredMethod("pid"); // only available in Java 9+
+        } catch (NoSuchMethodException e) {
+            pidMethod = null;
+        }
+        PID_METHOD = pidMethod;
```

Comment:
These two blocks of code are functionally similar but stylistically different in the names used.
I suggest making them similar, perhaps changing to use DESTROY_FORCIBLY_METHOD.

Reply:
That makes sense - I've renamed that field to this suggested name.